scripts/lint_ilyakhov.py

```
@@ -0,0 +1,640 @@
+URL_OR_CODE = re.compile(r"```.*?```|`[^`\n]+`|https?://\S+", re.S)
+
+COMMON_KNOWLEDGE = [
+    re.compile(r"\bкак\s+известно\b", re.I),
+    re.compile(r"\bне\s+секрет\b", re.I),
+    re.compile(r"\bни\s+для\s+кого\s+не\s+секрет\b", re.I),
+    re.compile(r"\bвсем\s+известно\b", re.I),
+    re.compile(r"\bобщеизвестно\b", re.I),
+]
+
+VERBAL_NUMBERING = [
+    re.compile(r"\bво-первых\b", re.I),
```

Ruff (0.16.1)

[warning] String contains ambiguous `о` (CYRILLIC SMALL LETTER O). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)